openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/FeedCardBody/FeedCardBodyV1.tsx

```diff
@@ -165,10 +168,18 @@ const FeedCardBodyV1 = ({
   return (
     <div
       className={classNames(
-        'feed-card-body bg-grey-5 p-sm rounded-6',
+        `feed-card-body  p-sm rounded-6 ${
+          feed.type !== ThreadType.Announcement && 'bg-grey-5'
```

Comment:
we can memoize this `feed.type !== ThreadType.Announcement` condition and use at other place as well, I can see similar condition use in the file.

Reply:
Done, have memoized this condition and reused it wherever necessary.